manim/scene/scene_file_writer.py

```diff
@@ -499,13 +520,20 @@ def open_partial_movie_stream(self, file_path=None):
             self.video_container = video_container
             self.video_stream = stream
 
-    def close_partial_movie_stream(self):
+            self.queue: Queue[tuple[int, PixelArray]] = Queue()
```

Comment:
Ah I made a small mistake here, my bad.
```suggestion
            self.queue: Queue[tuple[int, PixelArray | None]] = Queue()
```